src/public/index.html

```diff
@@ -9,37 +9,54 @@
   </head>
 
 
-    <body>
-      <header>
-        <h1>Something Awesome!</h1>
-      </header>
-      <nav>
-        <ul>
-          <li><a href="/about/">About</a></li>
-          <li> <a href="/contact/">Contact</a></li>
-        </ul>
-      </nav>
+  <body>
+    <header>
+      <div class="title">
+        <h1 class="plaintext-color">
+            Something Awesome! <br/>
+        </h1>
+      </div>
+    </header>
+      <div class="content-container page">
+        <div class="nav-width">
+          <nav class = 'red'>
```

Comment:
Also minor, but in HTML it's standard to use double quotes on element attribute values, and no spaces around the equal sign.